package.json

```diff
@@ -8,22 +8,36 @@
     "service"
   ],
   "license": "MIT",
-  "main": "./lib/tool-bar",
+  "main": "./dist/tool-bar",
   "repository": {
     "type": "git",
     "url": "https://github.com/suda/toolbar.git"
   },
   "scripts": {
     "lint": "eslint . --fix --ext js,json && remark *.md",
-    "test": "atom --test spec"
+    "test": "atom --test spec",
+    "clean": "shx rm -rf dist",
+    "debug": "npm run clean && shx cp -r lib dist && env NODE_ENV=development env BABEL_ENV=development babel dist --out-dir dist",
+    "dev": "npm run clean && env NODE_ENV=development env BABEL_ENV=production rollup -c -w",
+    "build": "npm run clean && env NODE_ENV=production env BABEL_ENV=production rollup -c",
+    "ppublish": "bash ./ppublish.sh",
```

Comment:
Does one need to manually execute this before publishing?

Not sure if that's the best solution. Have you considered using the [npm script](https://docs.npmjs.com/misc/scripts) `prepublishOnly` (which, for the record, I'm not 100% works with `apm`). Or maybe even `preinstall`, completely avoiding the need to include the `dist` folder in the project?

Reply:
Unfortunately, apm is not able to do that. This is the pattern I have used with all the other atom packages.